components/provisioner/internal/provisioning/input_converter.go

```diff
@@ -234,6 +235,9 @@ func (c converter) providerSpecificConfigFromInput(input *gqlschema.ProviderSpec
 		if input.OpenStackConfig.FloatingPoolName == nil {
 			input.OpenStackConfig.FloatingPoolName = util.PtrTo(OpenStackFloatingPoolName)
 		}
+		if input.OpenStackConfig.CloudProfileName == nil {
+			input.OpenStackConfig.CloudProfileName = util.PtrTo(OpenStackCloudProfileName)
+		}
```

Comment:
We could use [`OkOrDefault`](https://github.com/kyma-project/control-plane/blob/ee936771fbc59754f47cdf8c95f7db70d79e9a67/components/provisioner/internal/util/pointers.go#L26-L32) for both of those